core/src/dpdk/mempool.rs

```diff
@@ -68,6 +67,11 @@ impl Mempool {
     pub fn name(&self) -> &str {
         self.raw().name[..].as_str()
     }
+
+    #[cfg(feature = "metrics")]
+    pub fn stats(&self) -> super::MempoolStats {
```

Comment:
minor: why not just import in MempoolStats here?

Reply:
cuz then you have to feature gate the import. relatively minor but spreading all these `cfg` all over is a major pita. for one import I figured why not just skip the extra import. I can certainly go back to that if that's preferred.

Reply:
I figured, but was just thinking consistency. Not a biggie for me in terms of preference though.